src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs

```diff
@@ -355,7 +355,7 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
         {"dotnet_enable_brace_matching", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Brace Matching")},
         {"dotnet_enable_event_hook_up", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Event Hookup")},
         {"dotnet_format_on_save", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "FormatOnSave")},
-        {"dotnet_code_analysis_in_separate_process", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "OOP64Bit")},
+        {"dotnet_code_analysis_in_separate_process", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "OOP64Bit_New")},
```

Comment:
these flags still exist as an unsupported way to opt out of these features.  however, i've moved them to new regkeys so that everyone defualts into the supported state.  that way someone who disabled something in the past, isn't able to reenable if they want that.